src/scripts/lexmatch-sssom-compare.py

```diff
@@ -296,6 +296,14 @@ def get_unmapped_df(
     ]
 
     new_df = pd.concat([unmapped_lex_df, unmapped_mondo_df], axis=0)
+    if 'confidence' not in new_df.columns:
+        new_df['confidence'] = 0.5
```

Comment:
## Default to confidence 0.5
So, confidence ranges 0 to 1, 0.5 being "equally likely to be a match or not". This makes sense as a default, then.

### Questions
1. What are the consequences, if any, to setting this as a default value?
2. When you said earlier you were running a build with `-B`, it was this branch? To see if this bug fix worked?
3. It seems that (i) this fix will be wholly sufficient to address the new [AttributeError](https://github.com/monarch-initiative/mondo-ingest/pull/578#discussion_r1646653777) in `mondo-ingest`. And I know of (ii) the fix needed in `sssom-py`. Also, (iii), you wanted me to investigate why there are no confidence values being assigned. Other than these 3 things, are there any other tasks for us to complete as a result of dealing with this error?